deepchem/feat/test_oneHotFeaturizer.py

```diff
@@ -0,0 +1,19 @@
+from unittest import TestCase
```

Comment:
Could we rename this file `test_one_hot.py`? We haven't been using camel case in file names so far.